dll/win32/shell32/shlfileop.cpp

```diff
@@ -153,7 +153,7 @@ static INT_PTR ConfirmMsgBox_Init(HWND hDlg, LPARAM lParam)
 
     SetWindowTextW(hDlg, info->lpszCaption);
     ShowWindow(GetDlgItem(hDlg, IDC_YESTOALL_MESSAGE), SW_HIDE);
-    SetPropW(hDlg, L"WINE_CONFIRM", info->lpszText);
+    SetWindowLongPtrW(hDlg, DWLP_USER, (LONG_PTR)info->lpszText);
```

Comment:
Do you know since which commit this started to leak? Does your fix have a positive impact on the testbots?
Can I spot this leak by any debug-print on the testbots?

Reply:
For notice, the SetProp(W) here has always been here with us.

Reply:
Since it contains the name WINE I assume it has been like this since forever?

It should not impact the testbots unless they have problems with running out of global atoms.

You cannot spot these easily. They are however a shared resource and there can only be 0xC000? of them per winstation?/desktop? (there is an extension shipped with WinDbg that is supposed to list them but I can never get that to work but I wrote my own and I think [WindowsHookEx](https://github.com/learn-more/WindowsHookEx) can also show them).